docs/Deploy/single-sign-on-sso.md

```diff
@@ -1,75 +1,98 @@
 # Single sign-on (SSO)
 
-Learn how to configure single sign-on (SSO) and just-in-time (JIT) user provisioning.
+Fleet supports SSO and just-in-time (JIT) user provisioning using any identity provider (IdP) that supports SAML.
 
-## Overview
+Fleet supports both service (SP) initiated login and IdP initiated login.
 
-Fleet supports SAML single sign-on capability.
+To configure SSO, follow steps for your IdP and then complete [Fleet configuration](#fleet-configuration).
 
-Fleet supports both SP-initiated SAML login and IDP-initiated login. However, IDP-initiated login must be enabled in the web interface's SAML single sign-on options.
+## Okta
 
-Fleet supports the SAML Web Browser SSO Profile using the HTTP Redirect Binding.
+Create a new SAML app in Okta:
 
-> Note: The email used in the SAML Assertion must match a user that already exists in Fleet unless you enable [JIT provisioning](#just-in-time-jit-user-provisioning).**
+![Example Okta IdP Configuration](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/okta-idp-setup.png)
 
-## Identity provider (IDP) configuration
+If you're configuring [end user authentication](../Using%20Fleet/MDM-macOS-setup-experience.md#end-user-authentication-and-eula), use `https://<your_fleet_url>/api/v1/fleet/mdm/sso/callback` for the **Single sign on URL** instead.
 
-Setting up the service provider (Fleet) with an identity provider generally requires the following information:
+Once configured, you will need to retrieve the Issuer URI from the `View Setup Instructions` and metadata URL from the `Identity Provider metadata` link within the application `Sign on` settings. See below for where to find them:
```

Comment:
I think the things in backticks should be bold text instead (since they're UI elements not code names), and looks like we can lowercase "Issuer URI" (since it's not capitalized in the image):

```suggestion
Once configured, you will need to retrieve the issuer URI from **View Setup Instructions** and metadata URL from the **Identity Provider metadata** link within the application **Sign on** settings. See below for where to find them:
```